### DISCUSSION TITLE
Best way to persist data when redid is restarted/crashed

### DISCUSSION THREAD

**Discussion by BatuhanCakir** [taskforcesh/bullmq#1316] [General]
Hey,
I wanted to use BullMQ but a specification of my Project is that the data in the q will be persistent(not important that everything will be saved). Is there a way to do it because I couldn't find it in the docs but persistence is checked in the Readme.

With kind regards

**Comment by manast** [comment]
If you are new to Redis I recommend you read this document thoughtfully: https://redis.io/docs/manual/persistence/
Most Redis providers include also some kind of backup mechanism.
